aiohttp/client_reqrep.py

```diff
@@ -572,6 +583,7 @@ def __init__(self, method, url, *,
         self._timer = timer if timer is not None else TimerNoop()
         self._auto_decompress = auto_decompress
         self._cache = {}  # reqired for @reify method decorator
+        self._traces = traces or []
```

Comment:
The same: replace `or` with explicit check for `is None` please

Reply:
done